Backends/CLX-fb/package.lisp

```diff
@@ -72,7 +72,5 @@
 		#:sheet-xmirror
```

Comment:
Commit message: "Moving it into Backend/ part was a mistake. This is part of the core protocol and does not require any extra hacks."

What are "it" and "this"? This whole paragraph is hard to understand.

Reply:
rephrased.